packages/connect-voting-disputable/src/models/CollateralRequirement.ts

```diff
@@ -0,0 +1,23 @@
+import { CollateralRequirementData, IDisputableVotingConnector } from '../types'
+
+export default class CollateralRequirement {
+  #connector: IDisputableVotingConnector
+
+  readonly id: string
+  readonly voteId: string
+  readonly token: string
+  readonly actionAmount: string
+  readonly challengeAmount: string
+  readonly challengeDuration: string
+
+  constructor(data: CollateralRequirementData, connector: IDisputableVotingConnector) {
+    this.#connector = connector
```

Comment:
I wonder about having the `connector` object on every model. Maybe we should only include it on those models that implement queries/subscriptions requests?

Reply:
Yes, I wasn't sure about that, but it makes sense, we can have it only if needed